code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -18,7 +18,7 @@
 	if(((100 - total_burn) < config.health_threshold_dead) && stat == DEAD) //100 is the magic human max health number
 		ChangeToHusk()                                                      //BECAUSE NO ONE THOUGHT TO USE LIVING/VAR/MAXHEALTH I GUESS
 	if(species.can_revive_by_healing)
-		var/obj/item/organ/brain/B = internal_organs_by_name["brain"]
+		var/obj/item/organ/internal/brain/B = internal_organs_by_name["brain"]
 		if(B)

```

Comment:
Later, this check should probably search for all vital organs, and check that they are all alive and present in one manner or another.